.github/workflows/tests.yml

```diff
@@ -49,7 +49,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install -r requirements_test.txt
+          pip install --no-cache-dir -r requirements_test.txt
```

Comment:
made this change to unblock on tests. Some caching of packages resulting in stale versions/hashes. Seems fine to leave it permanently.